docs/src/ComponentsPage.js

```diff
@@ -120,16 +111,68 @@ const ComponentsPage = React.createClass({
 
   handleNavItemSelect(key, href) {
     window.location = href;
-    this.setActiveNavItem();
   },
 
   componentDidMount() {
-    this.setActiveNavItem();
+    this.afterSections = {};
```

Comment:
It's possible this would actually all be easier without React-Waypoint. Oh well.